user-guide/Advanced_Modules/DOM/DomHelper_class.md

```diff
@@ -88,6 +88,97 @@ while (pagingHelper.MoveToNextPage())
+
+#### Filtering
+
+To specify which `DomInstances` need to be retrieved from the database, you can build filters using the `FilterElement` structures.
+
+Start a filter by specifying the field you want to filter on using the `DomInstanceExposers` class. Once a field is selected, choose the comparer and provide a value. These filters can then be concatenated with AND and OR conditions.
+
+**Base properties:**
+
+|Field                |Type        |Supported Comparer |
+|---------------------|------------|-------------------|
+|Id                   |string      |Equal, NotEqual |
+|SectionDefinitionIds |List\<Guid> |Contains, NotContains |
+|SectionIds           |List\<Guid> |Contains, NotContains |
+|DomDefinitionId      |Guid        |Equal, NotEqual |
+|Name                 |string      |Equal, NotEqual, Contains, NotContains |
+|StatusId             |string      |Equal, NotEqual, Contains, NotContains |
+|LastModified         |DateTime    |LessThan, GreaterThan |
+|LastModifiedBy       |string      |Equal, NotEqual, Contains, NotContains |
+|CreatedAt            |DateTime    |LessThan, GreaterThan |
+|CreatedBy            |string      |Equal, NotEqual, Contains, NotContains |
+
+**Fields:**
+
+These are the supported comparers when filtering on `DomInstance` values using the `DomInstanceExposers.FieldValues.DomInstanceField()` exposer.
+
+|Field Type                |Supported Comparer |
+|--------------------------|-------------------|
+|string                    |Equal, NotEqual, Contains, NotContains |
+|double                    |Equal, NotEqual, LessThan, LessThanOrEqual, GreaterThan, GreaterThanOrEqual |
+|long                      |Equal, NotEqual, LessThan, LessThanOrEqual, GreaterThan, GreaterThanOrEqual |
+|TimeSpan                  |Equal, LessThan*, GreaterThan* |
+|Boolean                   |Equal |
+|DateTime                  |Equal, NotEqual, LessThan, GreaterThan |
+|GenericEnum (int)         |Equal, NotEqual, LessThan, LessThanOrEqual, GreaterThan, GreaterThanOrEqual |
+|List GenericEnum (int)    |Contains |
+|GenericEnum (string)      |Equal, NotEqual, Contains, NotContains |
+|List GenericEnum (string) |Contains |
+|Guid                      |Equals, NotEquals |
+|List Guid                 |Contains |
+|List string               |Contains |
+
+*Only supported for [STaaS](xref:STaaS)
+
+> [!NOTE]
+> From DataMiner 10.4.5/10.4.0 [CU2] onwards, `string` filters are handled as case-insensitive when using the OpenSearch database (which is how STaaS has handled `string` reads since its introduction). Prior to DataMiner version 10.4.5/10.4.0 [CU2], `string` filters are handled as case-sensitive.
```

Comment:
I switched the two sentences around, as we usually first mention the current behavior, followed by the behavior in older DataMiner versions. Is this sentence still OK content-wise? If so, the PR looks ready to merge.

Reply:
Looks great. Thanks!